docs/content/en/docs/concepts/support-versions.md

```diff
@@ -50,7 +50,7 @@ The following table notes which EKS Anywhere and related Kubernetes versions are
 | 0.6 | 1.21, 1.20                   | October 29, 2021 | No | 
 | 0.5 | 1.21, 1.20                   | September 8, 2021 | No | 
 
-* [Amazon EKS Anywhere Enterprise subscription](https://aws.amazon.com/eks/eks-anywhere/pricing/) is required to receive AWS support on any Amazon EKS Anywhere clusters.
+* [Amazon EKS Anywhere Enterprise Subscription](https://aws.amazon.com/eks/eks-anywhere/pricing/) is required to receive AWS support on any Amazon EKS Anywhere clusters.
```

Comment:
* [Amazon EKS Anywhere Enterprise Subscription](https://aws.amazon.com/eks/eks-anywhere/pricing/) is required to receive support for EKS Anywhere from AWS.